addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1676,7 +1681,18 @@ var SnippetEditor = Widget.extend({
                 this.dragState.currentHeight = (maxRowEnd - 1) * (gridProp.rowSize + gridProp.rowGap) - gridProp.rowGap;
             }
         }
-    }
+    },
+    /**
+     * Called when the mouse is released outside the page iframe (e.g. the
+     * editor panel). This is only useful in Chrome,  where the 'stop' event of
+     * jQuery Draggable does not trigger a 'mouseup' event outside of the
+     * "preview" page iframe.
+     *
+     * @private
+     */
+    _onParentWindowMouseup() {
+        this.ownerDocument.dispatchEvent(new Event('mouseup'));
+    },
```

Comment:
I think that in the theoretical case where there is no iframe (like before), this would create an infinite loop of mouseup? 
1. You set a mouseup listener on the window.parent, which is window in case there is no parent
2. On mouseup, you trigger a mouseup in the deepest iframe... which is the one where you add the mouseup listener
3. repeat, repeat, repeat, ...

Maybe marking the `addEventListener` with {once: true} would be nice. And on top of that instead of `window.parent`, you could ask the snippetMenu for its own window and only add those listener if it differs from the edition window (or maybe we actually have a dedicated parameter who already knows that info? Maybe check with ARD).

Did you tested in mass mailing? They have an iframe too but the editor is still inside if I am not mistaken, is it not a problem?

Reply:
`you trigger a mouseup in the deepest iframe` 
In fact, this "mouseup" will trigger the jQuery Draggable 'stop' event in which the `removeListener` is located, so it shouldn't be a problem. But I'm still going to do what you suggest as a precaution.

`Did you tested in mass mailing?`
Yes, in "mass mailing" the edit panel is also in the iframe indeed. There we had the same issue but releasing the mouse in the upper part of the viewport (the top window). That's why I used `window.parent` and not `window from the edit panel` (to fix it also in "mass mailing").

I will add '{once: true}' and complete the commit message to explain that the commit also fixes the problem in "mass mailing"

Reply:
@qsm-odoo changes are done